bin/client_build_xgo

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# Usage example:
```

Comment:
This is not an example - it is an interface.

```bash
# Usage:
#> bin/client_build_xgo <os>/<arch>
#
# Examples:
# Cross compile (Unix):
# bin/client_build_xgo linux/amd64
...
```

Reply:
Improved